src/pages/list-wizards.page.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  Component
+} from '@angular/core';
+import {archivedCharactersSelector} from '../selectors/form';
+import {
+  select,
+  NgRedux
+} from 'ng2-redux';
+import {Observable} from 'rxjs';
+import {ICharacter} from '../store/types/character';
```

Comment:
Could we import types from `store/types` in case we need to move things around again?